src/components/styles/PromptActionButtons.scss

```diff
@@ -1,8 +1,10 @@
 @import "../../styles/colors";
-.action-row {
-    height: 60px;
-    display: flex;
-    align-items: center;
+
```

Comment:
Thank u for fixing my jank css <3. Just make sure things look gucci in mobile/iPhone 6 size on `Get Involved` & `WiSE` pages. Thanks!